src/objectTranslations/Contact-ca.objectTranslation

```diff
@@ -3473,36 +3473,64 @@
         <name>PreferredPhone__c</name>
         <picklistValues>
             <masterLabel>Home</masterLabel>
-            <translation>Llar</translation>
+            <translation>Home Phone</translation>
```

Comment:
We should use the Catalán translation for all these, not English (available in the fields with the same name).

Reply:
I think since this is a Platform Only language some fields are not being translated, so I updated the picklist translations corresponding to those fields to English.

Reply:
They were previously translated though, right? So that might be a bit confusing. Is it because the Home, Mobile, and Other are not translated in the UI, and so if we translated the picklist, the functionality would still not work for them?

Reply:
Correct!

Reply:
Interesting. Thoughts on this one, @ceiroa? We're essentially taking away translations... but... this wouldn't be working for old or new customers anyway, since the actual fields themselves are not translated by Salesforce.

So, Catalan customers should translate both the Preferred Phone picklist and the standard field values if they need this to work in Catalan.